docs/tutorials/quickstart-app-auth.md

```diff
@@ -55,7 +53,17 @@ guest. Let's fix that now and add auth.
 
 # The Auth Configuration
 
-1. Open `app-config.yaml` and change it as follows
+A default Backstage installation includes multiple authentication providers out
+of the box. The steps to enable new authentication providers in Backstage are
+very similar to each other, the biggest difference is usually configuring the
+external authentication provider. Please see a subset of possible providers and
+instructions to integrate them below. Steps 1 & 2 are described separately for
+each provider and steps beyond that are common for all.
+
+<details><summary>GitHub</summary>
```

Comment:
Thank you for adding instructions for a bunch of providers!

The details/summary tags currently look a bit weird on the microsite, fixing that here: https://github.com/backstage/backstage/pull/4091